lib/propolis/src/vmm/machine.rs

```diff
@@ -486,7 +487,7 @@ impl<'a> MemCtx<'a> {
             .map
             .highest_addr(|entry| matches!(entry.kind, MapKind::SysMem(_, _)))?
             as u64;
-        Some(GuestAddr(lowest)..GuestAddr(highest))
+        Some(RangeInclusive::new(GuestAddr(lowest), GuestAddr(highest)))
```

Comment:
Purely a question, but does the `..=` syntax not work here? It may require implementing some trait that's otherwise noisy, so this is perfectly good to go; I really am just curious.

Reply:
Good call--it does work here, it just didn't occur to me to use it. Done.